client/rpcclient.go

```diff
@@ -67,6 +67,11 @@ func (c *FinalityGadgetGrpcClient) QueryBlockRangeBabylonFinalized(blocks []*typ
 		return nil, err
 	}
 
+	// No block in the range is babylon finalized
+	if res.LastFinalizedBlockHeight == 0 {
```

Comment:
how do you distinguish between "no block is finalized" and "the latest finalized block is block 0"?

Reply:
This value is only return from the [implementation](https://github.com/babylonlabs-io/finality-gadget/pull/20/files#diff-b2d01817caea21afda676b7e2393d0d8c983fb8c96a8d88f067e4ef2bab73ae1) of `rpcserver.go`, so it means "no block is finalized" here.